test/clj/election/controllers/votes_test.clj

```diff
@@ -0,0 +1,81 @@
+(ns election.controllers.votes-test
+  (:use clojure.test)
+  (:require
+    [election.controllers.votes :as v]
+    [clj-time.core :as t]
+    [clj-time.coerce :as c]
+    [election.i18n.messages :as i18n]
+  )
+)
+
+
+; (defn- check-election-start [election]
```

Comment:
You can probably get rid of those now

Reply:
I actually wanted to be thorough but it is true that this is very low level unit tests.

Reply:
Oops my bad. I though you were talking about the unit tests and not the comments.